Documentation/architecture.md

```diff
@@ -7,7 +7,7 @@ the kernel and gather the log data. Inspector Gadget's userspace
 utilities fetch the log data from ring buffers and display it. What BPF
 programs are and how Inspektor Gadget uses them is briefly explained here.
 
-![k8s integration](k8s.svg)
+![k8s integration](images/k8s.svg)
```

Comment:
Maybe `assets` rather than `images`? so then we can put there other stuff too like gifs, CSS and whatever we need :)

Reply:
yes, thanks